ansible/monitoring.yml

```diff
@@ -83,6 +89,7 @@
         - "flavor:c{{ ansible_processor_vcpus }}.m{{ (ansible_memtotal_mb/1024)|round|int }}"
         - "public_ipv4:{{ public_ipv4 }}"
         - "role:{{ node_role|default('unknown') }}"
+        - "network_id:{{ node_config.fork_management.network_id|default('unknown') }}"
```

Comment:
I think this will fail if there is no fork_management defined

Reply:
good catch, i will have same issue with backup template file.

Reply:
fixed in 7b0b3dd